src/Aspire.Dashboard/Model/ResourceEndpointHelpers.cs

```diff
@@ -41,7 +49,8 @@ public static List<DisplayedEndpoint> GetEndpoints(ResourceViewModel resource, b
         // - other urls
         // - endpoint name
         var orderedEndpoints = endpoints
-            .OrderByDescending(e => e.Url?.StartsWith("https") == true)
+            .OrderByDescending(e => e.SortOrder)
+            .ThenByDescending(e => e.Url?.StartsWith("https") == true)
```

Comment:
Maybe add scheme as a field to DisplayedEndpoint and use that here instead?

Reply:
Instead of the "https" check? This was already here :smile:

Reply:
Oops I didn't realize it was already there. Might as well change it regardless!

Reply:
I think this logic is just specifically looking to ensure https URLs are shown first so I don't think it requires the scheme to passed through separately. At this point we're just dealing with URL strings so there's no need to keep the scheme separate IMO.